src/portal/e2e/db-testcases.spec.ts

```diff
@@ -0,0 +1,287 @@
+import { test, expect } from '@playwright/test';
+
+async function createUser(page) {
```

Comment:
```suggestion
async function createUser(page): string {
```

do include type of the return value.